accelerator/cli_info.h

```diff
@@ -97,6 +98,7 @@ static struct ta_cli_argument_s {
     {"done_list", required_argument, NULL, DONE_LIST, "Set the value of `done_list_name`"},
     {"cache_capacity", required_argument, NULL, CACHE_CAPACITY, "Set the maximum capacity of caching server"},
     {"quiet", no_argument, NULL, QUIET, "Disable logger"},
+    {"start_file", required_argument, NULL, START_FILE, "A file to notify other process that TA has started."},
```

Comment:
I don't think it is meaningful to call the option "start_file." Can you explain more?